src/Compilers/CSharp/Test/Semantic/Semantics/RefEscapingTests.cs

```diff
@@ -45,23 +47,25 @@ static S1 MayWrap(ref int arg)
         }
     }
 ";
-            CreateStandardCompilation(text).VerifyDiagnostics(
-                // (21,42): error CS8168: Cannot return local 'local' by reference because it is not a ref local
+            CreateCompilationWithMscorlibAndSpan(text).VerifyDiagnostics(
```

Comment:
>CreateCompilationWithMscorlibAndSpan [](start = 12, length = 36)

That reminds me, how are we tracking the addition of Span to the libraries we reference in testing? so that we can remove this helper function

Reply:
I am not even sure we have to change this. Span implementations are platform-specific and we only care about the APIs that exposed. As long as we call right methods, we are producing right code.
I will enter a bug to add some tests with actual span (I think we can use slow-span, since that should be able to run on any platform), but that is not a priority. 

---
In reply to: [139844972](https://github.com/dotnet/roslyn/pull/22203#discussion_r139844972) [](ancestors = 139844972)

Reply:
[https://github.com/dotnet/roslyn/issues/22213](https://github.com/dotnet/roslyn/issues/22213)

---
In reply to: [139847810](https://github.com/dotnet/roslyn/pull/22203#discussion_r139847810) [](ancestors = 139847810,139844972)